### DISCUSSION TITLE
#solved

### DISCUSSION THREAD

**Discussion by Wileyfox-F1** [aahnik/tgcf#256] [Q&A]

**Comment by tissole** [comment]
What was your problem?

**Reply by Wileyfox-F1** [reply]
> What was your problem?

I thought past mode uses parallel cloning, so I doubted the config file. Then I realized TGCF uses step by step cloning according to the list. Nvm. Thanks for your interest in this topic.

**Reply by tissole** [reply]
Cloning multiple sources to the same destination would be very messy if done in parallel. The files will be mingled and lose their order. So I think that's why is done one after another.

**Reply by Wileyfox-F1** [reply]
> Cloning multiple sources to the same destination would be very messy if done in parallel. The files will be mingled and lose their order. So I think that's why is done one after another.

Yeah! The current settings is the most optimal one. Loved it! Btw did you know how to tweak the bot settings to ignore post-editing of the source file? And what are the default commands applied for livemode (Userbot)?

**Reply by tissole** [reply]
In the docs I see only delete_sync option for live mode
https://github.com/aahnik/tgcf/wiki/How-to-configure-tgcf-%3F

When set true, deleting files in the source is also deleting from the destination
But in the config file, I also see the option delete_on_edit, but I don't know how to set it, probably deletes files on the destination when modified in the source.

I can't find an option to ignore the edited files in source.
